pkg/cluster/internal/providers/docker/network.go

```diff
@@ -275,7 +275,6 @@ func isNetworkAlreadyExistsError(err error) bool {
 }
 
 // returns true if:
-// - err is nil
```

Comment:
upps :upside_down_face:

Reply:
I don't know why we included this in the comment, but IMHO the caller should be handling nil error vs non-nil error.